### PR TITLE
Updates the Microsoft.AspNetCore.Mvc.Core reference to a safer version.

### PR DESCRIPTION
**Pull request by vuqtran88**
## Description

Snyk scan reported the agent uses the version of Microsoft.AspNetCore.Mvc.Core that has security issue. Even though the agent references this package version, but actual version of the package that would be loaded is the version specified from the customer's application. The agent targets the lowest version for supportability purposes.

Since the AspNetCore wrapper targets netstandard2.0 so whatever the applications use the AspNetCore wrapper won’t be using the Microsoft.AspNetCore.Mvc.Core version lower than 2.0.0. Because of this, update the package reference is still a safe change to overcome Snyk scan.

### CODE REVIEW

File: src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AspNetCore/AspNetCore.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 	<ItemGroup>
 		<PackageReference Include="Microsoft.AspNetCore.Diagnostics.Abstractions" Version="2.0.0" />
-		<PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="1.1.3" />
+		<PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.0.0" />
 		<PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.0.0" />
```
Comment (JcolemanNR) [inline src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AspNetCore/AspNetCore.csproj:10]: This package was flagged by SNYK as well

**Review by JcolemanNR — CHANGES_REQUESTED** [review]
The SNYK alert flagged Microsoft.AspNetCore.Hosting@2.0.0 as well

File: src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AspNetCore/AspNetCore.csproj (hunk 1 above)
Comment (JcolemanNR) [reply src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AspNetCore/AspNetCore.csproj:10]: It looks like we were discussing different Snyk issues :)

**Review by JcolemanNR — APPROVED** [review]
If this still passes all integration tests, then I think it will be good to eliminate some Snyk issues!

**Review by michaelgoin — APPROVED** [review]

**Pull request merged**